Cargo.toml

```diff
@@ -57,7 +57,7 @@ esp-idf-svc = { version = "=0.48.1", default-features = false }
 espflash = { git = "https://github.com/viamrobotics/espflash.git", branch = "monitor-output" }
 futures = "0.3.28"
 futures-lite = "1"
-futures-rustls = "=0.22.0"
+futures-rustls = "0.24"
```

Comment:
Just curious why 0.24 and not the latest which is 0.26.

Reply:
it clashes with viam-rust-utils which is stuck in the past because of the bug with DTLS padding :P